mava/components/jax/building/datasets.py

```diff
@@ -98,7 +98,7 @@ def on_building_trainer_dataset(self, builder: SystemBuilder) -> None:
             postprocess=self.config.postprocess,
         )
 
-        builder.store.dataset_iterator = iter(dataset)
+        builder.store.dataset = iter(dataset)
```

Comment:
Not sure why this changed. Should I change it back to dataset_iterator?

Reply:
I think we need to get it back as it was, else we can make changes on the unit test that mention the dataset as dataset_iterator and it will be fine!

Reply:
I think because the develop branch was not updated because we make this change with the latest commit with dataset unit test

Reply:
Yes, please

Reply:
Will do. I can't seem to see other differences with the develop branch? @OmaymaMahjoub maybe we can have a chat about that other file that might be missing?

Reply:
I see the test file showed up now. I fixed it :+1: Let me know if everything looks good now. @AsadJeewa did the system train in your experiments?

Reply:
It runs on docker but when I run on venv I get 
`jaxlib.xla_extension.XlaRuntimeError: INTERNAL: Attempting to perform BLAS operation using StreamExecutor without BLAS support`. This is a GPU memory issue so not sure if it a localised issue